backends/gpu/tools/stub_codegen/generate.sh

```diff
@@ -34,3 +39,10 @@ for API in "hip" "rocblas" "rocsolver" "miopen" "hipfft"; do
    ./bazel-bin/backends/gpu/tools/stub_codegen/impl_codegen \
        $(dirname $0)/$API.json | clang-format > $(printf $SRC_PATH $API)
 done
+
+# Hiprtc is currently rolled up in hip shared library. 
```

Comment:
If it's going to be split up, would it make sense to already have a separate hiprtc_stub? It should be OK for two `LoadSymbol` functions to load the same shared object file. `llvm::sys::DynamicLibrary::getPermanentLibrary()` says "It is safe to call this function multiple times for the same library."

Reply:
> If it's going to be split up, would it make sense to already have a separate hiprtc_stub? It should be OK for two `LoadSymbol` functions to load the same shared object file. `llvm::sys::DynamicLibrary::getPermanentLibrary()` says "It is safe to call this function multiple times for the same library."

Thanks. I will split it up.